### ISSUE TITLE
[Enhancement] Install without CLI access

### ISSUE THREAD

**Issue by TheFax** [LycheeOrg/Lychee#1994]
In some web hosting servers **there is not** CLI access.
Installing Lichee 3 on these web servers was possible, but it became not possible with Lichee 4.

The request is: enable the possibility to install Lychee 4 in a server without CLI access.
OR
Improve the documentation in order to help customers having not CLI access.

**Comment by d7415** [comment]
Have you tried https://lycheeorg.github.io/docs/#from-release ?

You will need to make sure that e.g. only the `public` folder is accessible, but we have our own limitations from the framework and cannot anticipate every hosting environment.

**Comment by TheFax** [comment]
Yes, I tried.
I tried to install Lychee in my web hosting server in a subfolder like this:
h t t p s : / / xxx.xxx / lychee /
(spaces placed to avoid a hyperlink on this comment)

The results are:
1) on my web hosting is NOT possible to hide the root folder or changing its permissions. (xxx.xxx/lychee/) So the root folder remain visible.
2) if I try to browse the public folder (xxx.xxx/lychee/public/) only a white page appears. No errors, no popups, no errors also on the local js console. Only a completely white page. 

I have no idea where the issue is and how to solve it.
The guide you linked says "download and extract"... This is what I done.

Thank you to help me and to try to understand the issue.
